<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::BranchDialog</class>
 <widget class="QDialog" name="Git::Internal::BranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Branches</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="infoGroupBox">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="repositoryLabel">
        <property name="text">
         <string notr="true">Repository: Dummy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Re&amp;fresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="localBranchGroupBox">
     <property name="title">
      <string>Branches</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QTreeView" name="branchView">
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="addButton">
            <property name="text">
             <string>&amp;Add...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeButton">
            <property name="text">
             <string>&amp;Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="renameButton">
            <property name="text">
             <string>Re&amp;name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="checkoutButton">
            <property name="text">
             <string>&amp;Checkout</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="diffButton">
            <property name="text">
             <string>&amp;Diff</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logButton">
            <property name="text">
             <string>&amp;Log</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="mergeButton">
            <property name="text">
             <string>&amp;Merge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rebaseButton">
            <property name="text">
             <string>Re&amp;base</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="cherryPickButton">
          <property name="toolTip">
           <string>Cherry pick top commit from selected branch.</string>
          </property>
          <property name="text">
           <string>Cherry Pick</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trackButton">
          <property name="toolTip">
           <string>Set current branch to track the selected one.</string>
          </property>
          <property name="text">
           <string>&amp;Track</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>branchView</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>refreshButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Git::Internal::BranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
